aidants_connect_web/models.py

```diff
@@ -66,32 +64,33 @@ def get_current_mandats_for_usager(self, usager):
         :param usager:
         :return: a queryset of the current mandats with the usagers
         """
-        current_mandats = Mandat.objects.filter(usager=usager, aidant=self).exclude(
-            expiration_date__lt=timezone.now()
-        ).order_by("creation_date")
+        current_mandats = Mandat.objects.current() \
+            .filter(usager=usager, aidant=self).order_by("creation_date")
         return current_mandats
 
     def get_expired_mandats_for_usager(self, usager):
         """
         :param usager:
         :return: a queryset of the expired mandats with the usagers
         """
-        expired_mandats = Mandat.objects.filter(usager=usager, aidant=self).exclude(
-            expiration_date__gt=timezone.now()
-        ).order_by("creation_date")
+        expired_mandats = Mandat.objects.expired() \
```

Comment:
Je me demande si on ne pourrait pas juste mettre l'expression dans le return. Qu'en penses tu ?

Reply:
Et seconde pensée, est-ce qu'on peut l'écrire comme ça ? Est-ce que ça rendrait l'expression plus lisible ?
```suggestion
        expired_mandats = Mandat.objects.expired() \
            .filter(usager=usager, aidant=self) \
            .order_by("creation_date")
```